regression/pages/studio/import_course_page.py

```diff
@@ -0,0 +1,36 @@
+"""
+Import course page
+"""
+from edxapp_acceptance.pages.studio.import_export import (
+    ImportMixin
+)
+from edxapp_acceptance.pages.common.utils import click_css
+
+from regression.pages.studio.course_page_studio import CoursePageExtended
+from regression.pages import UPLOAD_FILE_DIR
+
+
+class ImportCoursePageExtended(ImportMixin, CoursePageExtended):
+    """
+    Extended course import page.
+    """
+    def upload_tarball(self, tarball_filename):
+        """
+        Upload a tarball to be imported.
+        """
+        asset_file_path = UPLOAD_FILE_DIR + "/" + tarball_filename
+        # Make the upload elements visible to the WebDriver.
+        self.browser.execute_script(
+            '$(".file-name-block").show();$(".file-input").show()'
+        )
+        # Upload the file.
+        self.q(css='input[type="file"]')[0].send_keys(asset_file_path)
+        # Upload the same file again. Reason behind this is to decrease the
+        # probability or fraction of times the failure occur. Please be
+        # noted this doesn't eradicate the root cause of the error, it
+        # just decreases to failure rate to minimal.
+        # Jira ticket reference: TNL-4191.
```

Comment:
Do we still need to run this twice? I see TNL-4191 has been resolved. Perhaps the same approach needs to be taken here (waiting for the click handler)?